### ISSUE TITLE
Ghost log file permissions

### ISSUE THREAD

**Issue by kirrg001** [TryGhost/Ghost-CLI#303]
```
> kate@dev-kate-01:~/ghost-kate$ ghost ls
┌────────────────┬──────────────┬──────────────┬──────────────────────┬──────────────────────────────────────┬──────┬─────────────────┐
│ Name           │ Location     │ Version      │ Status               │ Url                                  │ Port │ Process Manager │
├────────────────┼──────────────┼──────────────┼──────────────────────┼──────────────────────────────────────┼──────┼─────────────────┤
│ 207-154-219-31 │ ~/ghost-kate │ 1.0.0-beta.2 │ running (production) │ https://kate-ghost-test.acburdine.me │ 2368 │ systemd
```

```
> kate@dev-kate-01:~/ghost-kate$ ghost log
An error occurred.
Message: 'file does not exist'
```

```
> kate@dev-kate-01:~/ghost-kate$ ghost run
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: EACCES: permission denied, open '/home/kate/ghost-kate/content/logs/https___kate_ghost_test_acburdine_me_production.error.log'
    at Error (native)
```

I had no problem with Ghost log files when i installed/started Ghost with my root user.
`kate` is part of the sudoers group. The whole installation (nginx, ssl, mysql, systemd) worked well.

```kate@dev-kate-01:~/ghost-kate$ ll content/logs/
total 8
drwxrwxr-x 2 ghost ghost 4096 Jul  6 19:13 ./
drwxrwxr-x 7 ghost ghost 4096 Jul  6 19:13 ../
```

**Comment by acburdine** [comment]
The actual permissions error is because you tried to run `ghost run`, which won't work after the permissions have been changed by the ghost user.

However, the first issue of `ghost log` erroring when a log file doesn't exist, is a different one. I can fix that 😄

**Comment by kirrg001** [comment]
I am still hitting this permission error.
Even with your fix for `ghost log`.

```
> ghost start (or ghost run)

events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: EACCES: permission denied, open '/var/www/ghost/content/logs/https___kate_ghost_2_acburdine_me_production.error.log'
    at Error (native)
```

Permissions:
`drwxrwxr-x 2 ghost ghost`

**Using `sudo ghost run` or `sudo ghost start` resolves this.**
But i thought that is not needed? 😕

**Comment by kirrg001** [comment]
Can't reproduce anymore. Will add to the troubleshoot section.

**Comment by acburdine** [comment]
this should actually be fixed by #327
